cafebabel/core/routing.py

```diff
@@ -5,3 +5,9 @@ class RegexConverter(BaseConverter):
     def __init__(self, url_map, *items):
         super(RegexConverter, self).__init__(url_map)
         self.regex = items[0]
+
+
+class LangConverter(BaseConverter):
+    def __init__(self, url_map, *items):
+        super(LangConverter, self).__init__(url_map)
+        self.regex = '[a-z]{2}'
```

Comment:
A bit open, maybe reuse `ALLOWED_LANGUAGES` from django-migrations and make it a setting?

Reply:
Actually not that easy given that we are not within a context where the `current_app.config` is loaded. Let's keep it that way for now except if you have a better idea :)

Reply:
We can't read settings at this stage as we are not yet in the app context